Testreglar/2.4.3/Nett/nett-2.4.3a.json

```diff
@@ -0,0 +1,171 @@
+{
+	"namn": "Nett-2.4.3a Tastaturekkefølgje ivaretar meiningsinnhald og betening 2023",
+	"id": "nett-2.4.3a",
+	"testlabId": 471,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nn",
+	"kravTilSamsvar": "<p>For innhold som får fokus ved tastaturnavigasjon og må navigeres i en bestemt rekkefølge, er ett av følgende alternativ oppfylt:</p>\r\n<ul>\r\n<li>tastaturrekkefølge og visuell rekkefølge er lik eller</li>\r\n<li>tastaturrekkefølge og visuell rekkefølge er ulik, men det er likevel mulig å forstå og bruke innholdet</li>\r\n</ul>",
+	"side": "2.1",
+	"element": "3.1",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "2.3"
+		},
+		{
+			"title": "2.4"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		}
+	],
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Kva side testar du?",
+			"ht": "<p>Angi URL eller side-ID.</p>",
+			"type": "tekst",
+			"label": "URL/Side:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Får du synleg fokusmarkering ved tastaturnavigasjon?",
+			"ht": "<p>Slik testar du:</p>\n<ul>\n<li>Trykk sakte på tab-tasten for å navigere på nettsida med tastaturet.</li>\n<li>Følg nøye med på navigasjonen og informasjonen i eventuell statuslinje for å orientere deg på nettsida.</li>\n<li>Vi er avhengige av synleg fokusmarkør for å kunne gjennomføre testen. Synleg fokusmarkering blir testa på suksesskriterium 2.4.7.</li>\n</ul>",
```

Comment:
Vurder om "trykk sakte" er riktig formulering

Reply:
Dette er en formulering vi har brukt i tidligere testregler, samt flere oppdaterte testregler der vi bruker tastatur for å verifisere.